var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -34,17 +34,22 @@
 class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     """Perl 5 is a highly capable, feature-rich programming language with over
        27 years of development."""
+
     homepage = "http://www.perl.org"
     # URL must remain http:// so Spack can bootstrap curl
     url      = "http://www.cpan.org/src/5.0/perl-5.24.1.tar.gz"
 
-    version('5.24.1', '765ef511b5b87a164e2531403ee16b3c')
-    version('5.24.0', 'c5bf7f3285439a2d3b6a488e14503701')
-    version('5.22.2', '5767e2a10dd62a46d7b57f74a90d952b')
+    # Development releases
+    version('5.25.11', '37a398682c36cd85992b34b5c1c25dc1')
+
+    # Maintenance releases (recommended)
+    version('5.24.1', '765ef511b5b87a164e2531403ee16b3c', preferred=True)
+    version('5.22.3', 'aa4f236dc2fc6f88b871436b8d0fda95')
+
+    # End of life releases
     version('5.20.3', 'd647d0ea5a7a8194c34759ab9f2610cd')
-    # 5.18.4 fails with gcc-5
-    # https://rt.perl.org/Public/Bug/Display.html?id=123784
-    # version('5.18.4' , '1f9334ff730adc05acd3dd7130d295db')
```

Comment:
This release builds just fine for me with GCC 6. If you read the bug report, it sounds like the problem is that the tests don't pass. There are patches for this, but it sounds like it's a problem with the tests, not with Perl. Since it builds fine without tests and isn't the preferred version, I think it's fine to uncomment this.